modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -1505,14 +1505,15 @@ public String snakeCase(String name) {
     }
 
     /**
-     * Capitalize the string
+     * Capitalize the string. Please use org.openapitools.codegen.utils.StringUtils.camelize instead as this method will be deprecated.
      *
      * @param name string to be capitalized
      * @return capitalized string
+     * @deprecated
      */
     @SuppressWarnings("static-method")
     public String initialCaps(String name) {
-        return StringUtils.capitalize(name);
+        return camelize(name);
```

Comment:
camelize removes underscores which may cause issues with the InlineModelResolver and other places where underscores are being introduced. in the cpp rest sdk client generator this causes the class names to be different than the file names (some method calls have used initialCaps and some do not) which causes import errors.
I will open a dedicated issue for this.
Regards,
Omer.

Reply:
Yes, please open a separate issue with a (minimal) spec to reproduce the issue.